GannettDigital/api-docs
# ANALYTICS-4446 | update-branding

**References**: [ANALYTICS-4446](https://jira.gannett.com/browse/ANALYTICS-4446)

**Description**:
It's time for some rebranding. `LOCALiQ -> LocaliQ`  and `Reachlocal -> LocaliQ` , images have been updated and I changed a couple colors slightly to match the colors used on the new LocaliQ site. The only references to reachlocal left are the ReachPay sections and the references to the `organization` `reachlocal` that is currently cooked into a lot of our API responses. Making that change is outside the scope of this PR.

**To Test**:
Follow the README to spin up a local version of the application (dockerized or not) and then confirm API docs work as expected and all case of reach local have been updated.
